src/field_theory/splitting_field.lean

```diff
@@ -94,7 +95,6 @@ by rw [nat_degree_remove_factor, hfn, n.add_sub_cancel]
 
 /-- Auxiliary construction to a splitting field of a polynomial, which removes
 `n` (arbitrarily-chosen) factors.
-
```

Comment:
I assume this was accidental?

Reply:
well that's a github error I didn't expect to see, "applying suggestions on deleted lines is not currently supported". Done manually, hopefully this is the last thing necessary!